packages/components/Avatar/src/AvatarTokens.android.ts

```diff
@@ -0,0 +1,203 @@
+      fontSize: globalTokens.font.size500,
+      square: {
+        borderRadius: globalTokens.corner.radius60,
+      },
+    },
+    size72: {
+      size: 72,
+      badgeSize: 'large',
+      iconSize: 32,
+      fontSize: globalTokens.font.size500,
+      square: {
+        borderRadius: globalTokens.corner.radius60,
+      },
+    },
+    size96: {
+      size: 96,
+      badgeSize: 'extraLarge',
+      iconSize: 48,
+      fontWeight: globalTokens.font.weight.regular,
+      fontSize: globalTokens.font.size700,
+      square: {
+        borderRadius: globalTokens.corner.radius80,
+      },
+    },
+    size120: {
+      size: 120,
+      badgeSize: 'extraLarge',
+      iconSize: 48,
+      fontWeight: globalTokens.font.weight.regular,
+      fontSize: globalTokens.font.size900,
+      square: {
+        borderRadius: globalTokens.corner.radius80,
+      },
+    },
+    neutral: {
+      color: t.colors.neutralForeground3,
+      iconColor: t.colors.neutralForeground3,
+      ringColor: t.colors.transparentStroke,
+    },
+    brand: {
+      backgroundColor: t.colors.brandBackground,
+      color: t.colors.neutralForegroundOnColor,
+      iconColor: t.colors.neutralForegroundOnBrand,
+      ringColor: t.colors.brandStroke1,
+    },
+     brandInverted: {
+      backgroundColor: t.colors.neutralBackground1,
+      iconColor: t.colors.brandForeground1,
+      ringColor: t.colors.brandStroke1,
+    },
+    accent: {
+      backgroundColor: t.colors.brandBackgroundTint,
+      iconColor:t.colors.brandForegroundTint,
+      ringColor: t.colors.brandStroke1,
+    },
+    darkRed: getColorProps('darkRed', t),
+    cranberry: getColorProps('cranberry', t),
+    red: getColorProps('red', t),
+    pumpkin: getColorProps('pumpkin', t),
+    peach: getColorProps('peach', t),
+    marigold: getColorProps('marigold', t),
+    gold: getColorProps('gold', t),
+    brass: getColorProps('brass', t),
+    brown: getColorProps('brown', t),
+    forest: getColorProps('forest', t),
+    seafoam: getColorProps('seafoam', t),
+    darkGreen: getColorProps('darkGreen', t),
+    lightTeal: getColorProps('lightTeal', t),
+    teal: getColorProps('teal', t),
+    steel: getColorProps('steel', t),
+    blue: getColorProps('blue', t),
+    royalBlue: getColorProps('royalBlue', t),
+    cornflower: getColorProps('cornflower', t),
+    navy: getColorProps('navy', t),
+    lavender: getColorProps('lavender', t),
+    purple: getColorProps('purple', t),
+    grape: getColorProps('grape', t),
+    lilac: getColorProps('lilac', t),
+    pink: getColorProps('pink', t),
+    magenta: getColorProps('magenta', t),
+    plum: getColorProps('plum', t),
+    beige: getColorProps('beige', t),
+    mink: getColorProps('mink', t),
+    platinum: getColorProps('platinum', t),
+    anchor: getColorProps('anchor', t),
+    burgundy: getColorProps('burgundy', t),
+    hotPink: getColorProps('hotPink', t),
+    orchid: getColorProps('orchid', t),
+  } as AvatarTokens);
```

Comment:
Are there only those colors which are used in android? Are there new colors?

Reply:
There are 3 colors I removed as they are not in Android designs, but no additional colors.